### PR TITLE
WSL in README.md #670

### PR DESCRIPTION
**Pull request by scdivi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ To view which version of Nutils is currently installed, run:
 
     python3 -m pip show nutils
 
+Nutils can be installed in a Windows machine using WSL environment. If you want to assemble matrices in parallel using nutils on a Windows machine, then WSL is the way to go. Instructions to setup WSL are available [here](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-10#1-overview). After setting up WSL, nutils can be installed using the above instructions.
```
Comment (joostvanzwieten) [inline README.md:50]: The linked tutorial mentions an [updated version](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#1-overview) that also works for Windows 11. Can you verify that the updated tutorial works as well? If so, please update the link.

Comment (joostvanzwieten) [inline README.md:50]: This document uses numbered references and a maximum line width of 80 characters for running text. Could you please use the same style. I'm happy to do this for you if you [allow maintainers to edit your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Review by joostvanzwieten — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scdivi) [reply README.md:50]: The updated tutorial works as well.

**Comment by joostvanzwieten** [comment]
Thanks for working on this, Sai!

File: README.md (hunk 1 above)
Comment (scdivi) [reply README.md:50]: The updated tutorial works as well. I will update the link.

Comment (scdivi) [reply README.md:50]: Ah! I get it. Will follow the same.

**Comment by joostvanzwieten** [comment]
Thanks for the updates. Could you rebase and squash the commits on top of the current master? Assuming you have a local repository with remote `origin` pointing to this `github.com/evalf/nutils.git` and a local branch named `patch-1` with the contents of this PR, run in a terminal:

```sh
git fetch origin
git checkout patch-1
git rebase -i origin/master
```

This will open an editor with the following contents:

```
pick 1a5f8c22 WSL in README.md #670
pick 391e07c5 update WSL in README.md #670
pick 602ada69 numbered reference to WSL in README.md #670a

# a bunch of comments
```

Change all but the first `pick`s into `f`s,

```
pick 1a5f8c22 WSL in README.md #670
f 391e07c5 update WSL in README.md #670
f 602ada69 numbered reference to WSL in README.md #670

# a bunch of comments
```

and close the editor. Then push the updates to *your* repository.

I can also do this for you if you [allow me to edit your branch associated with this PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Review by joostvanzwieten — APPROVED** [review]

**Pull request merged**